python/build_all/linux/build.sh

```diff
@@ -31,8 +31,7 @@ process_args()
 process_args $*
 
 # instruct C builder to include python library and to skip tests
-
-./c/build_all/linux/build.sh --build-python $PYTHON_VERSION --skip-unittests $*
+./c/build_all/linux/build.sh --build-python $PYTHON_VERSION --skip-unittests --skip-samples --skip-service-client $*
```

Comment:
@tameraw this is for the python client library though. I would assume if you are running the build script from here, you want to focus on the python outputs only? It also reduces the compile requirements when building on smaller devices i.e. raspberry pi as we only go on to call `make` so all targets that are set are built.

It is still defaults to build samples and service client when running the script from the 'c' build_all folder.